flickity/flickity.d.ts

```diff
@@ -0,0 +1,410 @@
+// Type definitions for Flickity v1.1.1
+// Project: http://flickity.metafizzy.co/
+// Definitions by: Chris McGrath <https://www.github.com/clmcgrath>
+// Definitions: https://github.com/clmcgrath/
+
+
+interface JQuery {
+    /**
+    * initialize fickity plugin
+    */
+    flickity: IFlickityJquery;
+}
+
+interface IFlickityJquery {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.